android/quest/src/main/assets/configs/app/profiles/default_profile_config.json

```diff
@@ -1231,7 +1231,7 @@
                   "id": "availableTasks",
                   "relatedResourceId": "availableTasks",
                   "resourceType": "Task",
-                  "conditionalFhirPathExpression": "Task.reasonCode.coding[0].code = 'immunization_at_birth'"
+                  "conditionalFhirPathExpression": "Task.groupIdentifier.value = '0_d' and Task.executionPeriod.start > today() + 7 days"
```

Comment:
I think this timings should be `today() - 7days`. We want to show the tasks 7 days before they are DUE.